service/src/main/java/com/derpgroup/derpwizard/resource/HomeResource.java

```diff
@@ -37,7 +37,7 @@
  * @since 0.0.1
  */
 @Path("/")
-@Produces(MediaType.APPLICATION_JSON)
+@Produces(MediaType.TEXT_PLAIN)
```

Comment:
I see. We could change the return type to something that must marshal as a JSON object, e.g. `Map<String, Object>` instead:

`return ImmutableMap.of("message", "Hello, World!");`